products/urls.py

```diff
@@ -1,4 +1,7 @@
-from django.urls import path
+from django.urls  import path
+
+from .views       import ProductsView
 
 urlpatterns = [
+    path("/catalog", ProductsView.as_view()),
```

Comment:
```suggestion
    path("", ProductsView.as_view()),
```
8000/products 까지만 해도 충부히 restful 할 것 같습니다.
이 부분만 수정하면 이제 될 것 같아요.